target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-mr6350.dts

```diff
@@ -0,0 +1,136 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qcom-ipq4019-xx8300.dtsi"
+
+/ {
+	model = "Linksys MR6350";
+	compatible = "linksys,mr6350", "qcom,ipq4019";
+
+/*
+	reserved-memory {
+		#address-cells = <1>;
+		#size-cells = <1>;
+		ranges;
+
+		ramoops@87500000 {
+			compatible = "ramoops";
+			reg = <0x87500000 0x600000>;
+			console-size = <0x200000>;
+			pmsg-size    = <0x200000>;
+			record-size  = <0x200000>;
+			ecc-size = <16>;
+		};
+	};
+*/
+	
+	aliases {
+		led-boot = &led_blue;
+		led-failsafe = &led_red;
+		led-running = &led_green;
+		led-upgrade = &led_blue;
+		serial0 = &blsp1_uart1;
+	};
+
+	// Top panel LEDs, above Linksys logo
+	leds {
+		compatible = "gpio-leds";
+		
+		led_blue: blue {
+			label = "blue:panel";
+			gpios = <&tlmm 46 GPIO_ACTIVE_LOW>;
+		};
+
+		led_red: red {
+			label = "red:panel";
+			gpios = <&tlmm 47 GPIO_ACTIVE_HIGH>;
+			panic-indicator;
+		};
+
+		led_green: green {
+			label = "green:panel";
+			gpios = <&tlmm 49 GPIO_ACTIVE_HIGH>;
+		};
+
+
+		// On back panel, above USB socket
+
+		led_usb: usb {
+			label = "green:usb";

```

Comment:
```suggestion
		led-usb {
			color = <LED_COLOR_ID_GREEN>;
			function = LED_FUNCTION_USB;
```
(label is not used anywhere)

Reply:
@raenye: Thanks for the review. I fixed the findings